c2corg_ui/templates/route/edit.html

```diff
@@ -289,8 +289,19 @@ <h2 class="heading show-phone"><span translate>numbers</span></h2>
               </div>
             </div>
 
+            ## DIFFICULTIES DURATIONS
+            <div class="form-group half" ng-class="{ 'has-success': route.duration_difficulties}">
+              <label><span translate>duration_difficulties</span> (<span translate>days</span>)</span></label>
+               <select class="form-control" ng-model="route.duration_difficulties">
+                 <option value=""></option>
+                 <option value="5"><span>5min</span></option>
+                 <option value="60"><span>1h</span></option>
+                 <option value="360"><span>6h</span></option>
```

Comment:
line 44 of https://docs.google.com/spreadsheets/d/1p7-RakklDot9gFnciEGox75833zU0ZuC5pRAoLZKziY/edit#gid=1418510514